tests/unit/docker/test_hub_build.py

```diff
@@ -66,13 +76,15 @@ def test_hub_build_push():
     assert manifests[0]['name'] == summary['manifest_info']['name']
 
 
-@pytest.mark.skip(reason='Non reproducible error and super flaky in github. Please repair it.')
-def test_hub_build_push_push_again():
+def test_hub_build_push_push_again(monkeypatch, access_token_github):
```

Comment:
this should be moved to an integration test and not a unittest

Reply:
also the one pushing

Reply:
moved under `test_hub_usage` integration test